src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/brothers.html.twig

```diff
@@ -0,0 +1,75 @@
+{#**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="card" id="orderBrothersPanel">
+  <div class="card-header">
+    <h3 class="card-header-title">
+      {{ 'Linked orders'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      <span class="badge badge-primary rounded">{{ orderForViewing.brothers.brothers|length }}</span>
+    </h3>
+  </div>
+
+  <div class="card-body">
+    <table class="table mb-3">
+      <thead>
+      <tr>
+        <th>
+          {{ 'Order no. '|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
Is it the order ID? If yes, I'd rather write _Order ID_ instead, still localized in Admin.Orderscustomers.Feature. By the way, there is an extra space at the end of the string. ;-)

Reply:
All these translations comes from legacy page. I copied the translations just like they used to be.

I can change them however if you think it's better 😄

Reply:
Indeed it's an Order ID that is showed. And I have no idea what is this extra space for 😅

Reply:
Yep, it'd be more accurate with _Order ID_ instead, in _Admin.Orderscustomers.Feature_, thanks @matks!

Reply:
Done in https://github.com/PrestaShop/PrestaShop/pull/19704/commits/8a996d84c86bdb2923f42daf0124f04d402f5d80